### PR TITLE
data: Omdeshkar

### PR DESCRIPTION
**Pull request by Omdeshkar**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4706"></a>

### CODE REVIEW

File: data/Omdeshkar.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/OmDeshkar3",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Cheak out my Github",
+      "url": "https://github.com/Omdeshkar",
+      "icon": "FaGithub"
+    },
```
Comment (swagataroy30) [inline data/Omdeshkar.json:15]:
```suggestion
    }
```

**Review by swagataroy30 — COMMENTED** [review]
Can you please remove the comma at line 15. Thank You!

File: data/Omdeshkar.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "type": "personal",
+  "bio": " 19 |Learning front end dev | Open source newbie",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/OmDeshkar3",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Cheak out my Github",
```
Comment (Hrishi5111998) [inline data/Omdeshkar.json:12]: there is typo at line 12 , it should be **check** .

**Review by Hrishi5111998 — CHANGES_REQUESTED** [review]

File: data/Omdeshkar.json (hunk 2 above)
Comment (kumarsonsoff3) [reply data/Omdeshkar.json:12]:
```suggestion
      "name": "Check out my Github",
```

**Pull request merged**